template/lambda-sqs-worker-cdk/infra/appStack.ts

```diff
@@ -52,11 +53,14 @@ export class AppStack extends Stack {
 
     const architecture = '<%- lambdaCdkArchitecture %>';
 
-    const worker = new aws_lambda.Function(this, 'worker', {
+    const worker = new aws_lambda_nodejs.NodejsFunction(this, 'worker', {
       architecture: aws_lambda.Architecture[architecture],
-      code: new aws_lambda.AssetCode('./lib'),
+      entry: './src/app.ts',
+      bundling: {
+        sourceMap: true,
+        target: 'node20',
+      },
       runtime: aws_lambda.Runtime.NODEJS_20_X,
-      handler: 'app.handler',
```

Comment:
>Alternatively, an entry file and handler can be specified:

new nodejs.NodejsFunction(this, 'MyFunction', {
  entry: '/path/to/my/file.ts', // accepts .js, .jsx, .cjs, .mjs, .ts, .tsx, .cts and .mts files
  handler: 'myExportedFunc', // defaults to 'handler'
});

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda_nodejs-readme.html#configuring-esbuild